docs/using/specifics.md

```diff
@@ -143,7 +143,7 @@ sdf_len(sc, 100, repartition = 1) %>%
 Spylon kernel instantiates a `SparkContext` for you in variable `sc` after you configure Spark
 options in a `%%init_spark` magic cell.
 
-```python
+```text
```

Comment:
It would be preferable to have `python` remain here even though its not exactly Python syntax highlighting we would need - as this includes jupyter magic `%%` stuff.

Will it error if you go with `python` / did it automatically declare this to be `text` because it found it to be invalid python?

Reply:
Yes, it will error :(
I actually found `--skip-errors` flag which is what we want.
And right now ipython magic is not supported and there is already an upstream issue.
https://github.com/asottile/blacken-docs/issues/127

I think using this flag is the best solution here.